std_msgs/String type #“crosswalk” “busstop” “obstacle” “park_slot”
geometry_msgs/Point[] points
int32 id
